src/GameStateMatch.cpp

```diff
@@ -25,16 +25,63 @@ bool GameStateMatch::load() {
 	this->level = new Level();
 	if (!this->level->levelTexture.loadFromFile("assets/texture/checkerboard.png", this->game->renderer)) {
 		printf("Failed to load the level texture!\n");
+		success = false;
 	} else {
-		this->level->levelTexture.setDimensions(2000, 2000);	
+		this->level->levelTexture.setDimensions(2000, 2000);
 	}
-	
+
+	unsigned int playerMarineID = GameManager::instance()->createMarine();
+
+	// Create Dummy Marines
+	success = GameManager::instance()->createMarine(this->game->renderer, 1500, 1500);
+
+
+	Turret* dumbTurret = GameManager::instance()->getTurret(GameManager::instance()->createTurret());
+	if (!dumbTurret->texture.loadFromFile("assets/texture/turret.png", this->game->renderer)) {
+		printf("Failed to load the player texture!\n");
+		success = false;
+	}
+
+	Zombie* dumbZombie = new Zombie();
+	GameManager::instance()->addZombie(dumbZombie);
+	if (!dumbZombie->texture.loadFromFile("assets/texture/zombie.png", this->game->renderer)) {
+		printf("Failed to load the player texture!\n");
+		success = false;
+	}
+	dumbZombie = new Zombie();
```

Comment:
Lines 51 - 56 are redundant. You are assigning a new pointer to an already existing object and assigning it the exact same texture.

This is also a memory leak as the new object pointer is never freed.